frameworks/hdfs/docs/security.md

```diff
@@ -157,9 +156,27 @@ Install the DC/OS Apache HDFS service with the following options in addition to
 
 ## Authorization
 
-The DC/OS Apache HDFS service supports HDFS's native authorization primitives. If Keberos is enabled as detailed [above](#kerberos-authentication), then
+The DC/OS Apache HDFS service supports HDFS's native authorization, which behaves similarly to UNIX file permissions. If Keberos is enabled as detailed [above](#kerberos-authentication), then Kerberos principals are mapped to OS users which HDFS uses to assign permissions internally.
 
 ### Enable Authorization
 
 #### Prerequisites
 - Completion of  [Kerberos authentication](#kerberos-authentication) above.
+
+#### Set Kerberos Principal to User Mapping
+
+A custom mapping can be set to map Kerberos principals to OS user names. This is supplied by setting the parameter
+```
+{
+    "hdfs": {
+        "security_auth_to_local": "<custom mapping>"
+    }
+}
+```
+where `<custom mapping>` is a base64 encoded string. The mapping is base64 encoded to ensure it is sent to the service correctly.
```

Comment:
"base64-encoded" (missing hyphen) (twice).

Also, it's not clear what is meant by the second sentence.